spec.html

```diff
@@ -1095,15 +1095,37 @@ <h1>Identity</h1>
   </emu-clause>
 </emu-clause>
 
-<emu-clause id="sec-ecmascript-data-types-and-values" aoid="Type">
+<emu-clause id="sec-ecmascript-data-types-and-values">
   <h1>ECMAScript Data Types and Values</h1>
   <p>Algorithms within this specification manipulate values each of which has an associated type. The possible value types are exactly those defined in this clause. Types are further subclassified into ECMAScript language types and specification types.</p>
-  <p>Within this specification, the notation “Type(_x_)” is used as shorthand for “the <em>type</em> of _x_” where “type” refers to the ECMAScript language and specification types defined in this clause.</p>
 
   <emu-clause id="sec-ecmascript-language-types">
     <h1>ECMAScript Language Types</h1>
     <p>An <dfn variants="ECMAScript language types">ECMAScript language type</dfn> corresponds to values that are directly manipulated by an ECMAScript programmer using the ECMAScript language. The ECMAScript language types are Undefined, Null, Boolean, String, Symbol, Number, BigInt, and Object. An <dfn variants="ECMAScript language values">ECMAScript language value</dfn> is a value that is characterized by an ECMAScript language type.</p>
 
+    <emu-clause id="sec-type-ao" type="abstract operation">
+      <h1>
+        Type (
```

Comment:
I was thinking this AO would be better placed in section 7 "Abstract Operations". (Note that its first use occurs in 7.2.9.) Now I'm less sure, but I thought I'd mention it anyway.

Reply:
I think I like it here. Section 7 AOs should probably be dispersed to relevant sections.